server/.eslintrc.json

```diff
@@ -12,6 +12,8 @@
     "indent": ["error", 2],
     "linebreak-style": ["error", "unix"],
     "quotes": ["error", "single"],
-    "semi": ["error", "always"]
+    "semi": ["error", "always"],
+    "no-console": ["off"],
+    "no-unsafe-finally": ["off"]
```

Comment:
Also curious about this change?

Reply:
@lauraGgit we have return code in `finally` that is not allowed by the linter